user snippets/getUrlParameter.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Run a small snippet of JavaScript during a mabl flow/journey
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+  let url_string = document.URL; // gets the current URL
+  let url = new URL(url_string);
+  let Id = url.searchParams.get("Id");// give the paramter to retrieve
```

Comment:
You could make this more general purpose with a variable, e.g., `queryParamName` and changing the last two lines to

```
let value = url.searchParams.get(mablInputs.variables.user.queryParamName);
callback(value);
```

Reply:
Sure @srvance ,I implement the changes and updates it